server/src/api/utils.rs

```diff
@@ -19,7 +19,7 @@ pub fn format_address(
 ) -> String {
     format!(
         "{}, {} {}, {}",
-        address_line, address_city, address_postalcode, address_country
+        address_line.split(" ").skip(1).collect::<Vec<&str>>().join(" "), address_city, address_postalcode, address_country
```

Comment:
I noticed this earlier but yeah there's some annoying whitespace that I forgot to trim in the listings request. This unfortunately has the side effect of skipping the first word if there is no whitespace :upside_down_face:.

Reply:
Then can the number portion of `address_line` even be removed for certain?
with this kinda free form text input it's kinda lame

Reply:
It can, but consider that not all addresses start with a number line. You'd turn the string into a char iterator and then try parsing a number from there. If it fails, then turn the char iterator back into a string. If it succeeds, consume whitespace and form a new string with the remaining chars. As you can see, this is a pain :)

Reply:
Also, we can't remove the number portion because then addresses like 205 Albert Street make no sense ... where on Albert street? It's pretty long ...

Reply:
right right, not sure what to do about the address then. Maybe just give back postal code or something
the map using rounded latitude and longitude would give a rough location though